pkg/expr/nodes.go

```diff
@@ -10,10 +10,15 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/data"
 	"github.com/grafana/grafana/pkg/expr/classic"
 	"github.com/grafana/grafana/pkg/expr/mathexp"
+	"github.com/grafana/grafana/pkg/infra/log"
 
 	"gonum.org/v1/gonum/graph/simple"
 )
 
+var (
+	logger = log.New("expr.log")
```

Comment:
Maybe was too fast approving.

```suggestion
	logger = log.New("expr")
```